src/coreclr/vm/ClrEtwAll.man

```diff
@@ -211,6 +211,7 @@
                           value="8" eventGUID="{561410f5-a138-4ab3-945e-516483cddfbc}"
                           message="$(string.RuntimePublisher.ContentionTaskMessage)">
                         <opcodes>
+                            <opcode name="AwareLockCreated" message="$(string.RuntimePublisher.AwareLockCreatedOpcodeMessage)" symbol="CLR_AWARELOCK_CREATED_OPCODE" value="10"> </opcode>
```

Comment:
Maybe a new opcode is not necessary, the `win:Start` opcode can be used instead

Reply:
I think win:Info is a good choice here since we are not starting an action, and there won't be a corresponding stop.